ui/package.json

```diff
@@ -140,9 +140,5 @@
         "wdio-chromedriver-service": "8.0.1",
         "webdriver-manager": "12.1.8",
         "webdriverio": "8.0.13"
-    },
-    "overrides": {
-        "got": "^12.1.0",
-        "qs": "^6.10.3"
```

Comment:
Ideally we shouldn't have any package overrides which interrupt with the nested package dependencies. I've tested on my local with these packages removed, everything works as expected. Will double check and test on our other environments once merged